packages/api/src/account-lcd.ts

```diff
@@ -28,6 +35,23 @@ export default class Account extends LCDClient {
     return deriveMnemonic(mnemonic, path).privateKey
   }
 
+  static generateMnemonic(): string {
+    const entropy = randomBytes(32)
+    if (entropy.length !== 32) throw Error(`Entropy has incorrect length`)
```

Comment:
for some reason, I saw that test on the lunie library, it might be related to the fact that they run it in the browser or something like that. I didn't have any issues with node but I just kept the same test in case there are things that I don't know about it